### PR TITLE
Fixed typo on the sample code.

### PR DESCRIPTION
**Pull request by wuerges**
Fixed typo on the sample code:
`random_32-bytes` -> `random_32_bytes`

### CODE REVIEW

File: docs/basic-features/key_management.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ A `KeysManager` can be constructed simply with only a 32-byte seed and some inte
 let mut random_32_bytes = [0; 32];
 // Fill in random_32_bytes with secure random data, or, on restart, reload the seed from disk.
 let start_time = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
-let keys_interface_impl = lightning::chain::keysinterface::KeysManager::new(random_32-bytes, start_time.as_secs(), start_time.subsec_nanos());
+let keys_interface_impl = lightning::chain::keysinterface::KeysManager::new(random_32_bytes, start_time.as_secs(), start_time.subsec_nanos());
```
Comment (jkczyz) [inline docs/basic-features/key_management.md:15]: Now that I see this, I think this should actually be `&random_32_bytes`. Would you mind updating this while you're here?

Comment (ConorOkus) [inline docs/basic-features/key_management.md:15]:
```suggestion
let keys_interface_impl = lightning::chain::keysinterface::KeysManager::new(&random_32_bytes, start_time.as_secs(), start_time.subsec_nanos());
```

**Comment by jkczyz** [comment]
@wuerges Looks like there is a merge conflict. Could you rebase?

**Comment by wuerges** [comment]
Sure, don't worry.

**Comment by wuerges** [comment]
Sorry I'm not used to this workflow. Please let me know if I made a mistake.

**Comment by ConorOkus** [comment]
Looks good @wuerges , thanks for the fix

**Pull request merged**